stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/plugin/adapter/ChatClientReferenceAdapter.kt

```diff
@@ -0,0 +1,9 @@
+package io.getstream.chat.android.offline.plugin.adapter
+
+import io.getstream.chat.android.client.ChatClient
+import io.getstream.chat.android.client.api.models.QueryChannelsRequest
+import io.getstream.chat.android.offline.querychannels.QueryChannelsReference
+
+public class ChatClientReferenceAdapter(private val chatClient: ChatClient) {
```

Comment:
here we will add all requests that available currently in `ChatClient` and wrap them into `QueryReference<T,S>`